packages/headless/src/features/facets/range-facets/generic/range-facet-insight-utils.ts

```diff
@@ -21,15 +17,3 @@ export const getInsightAnalyticsActionForToggleRangeFacetSelect = (
     ? logFacetDeselect(payload)
     : logFacetSelect(payload);
 };
-
-export const getInsightAnalyticsActionForToggleRangeFacetExclude = (
-  facetId: string,
-  selection: RangeFacetValue
-) => {
-  const facetValue = `${selection.start}..${selection.end}`;
-  const payload: FacetSelectionChangeMetadata = {facetId, facetValue};
-
-  return isRangeFacetValueExcluded(selection)
-    ? logFacetDeselect(payload)
-    : logFacetExclude(payload);
-};
```

Comment:
This function was added by mistake during the implementation of facet exclusion. It is never actually used.